src/os/bluestore/Compression.cc

```diff
@@ -56,19 +56,34 @@ void Estimator::cleanup()
   uncompressed_size = 0;
   compressed_occupied = 0;
   compressed_size = 0;
+  compressed_area = 0;
   total_uncompressed_size = 0;
   total_compressed_occupied = 0;
   total_compressed_size = 0;
   actual_compressed = 0;
   actual_compressed_plus_pad = 0;
   extra_recompress.clear();
+  single_compressed_blob = true;
+  last_blob = nullptr;
 }
```

Comment:
The above lacks target_blob_size reset...

Reply:
>IIUC that's not a bitwise op. Hence IMO using bitwise &= operator is rather confusing
Correct, addressed from Ronen's review.

Reply:
`target_blob_size` no longer exists. `wctx` is reset.